### PR TITLE
Add doc on active replication and how to use CLI to set replication

### PR DESCRIPTION
**Pull request by apc999**
* Added the documentation on passive replication vs active replication 
* Updated docs on how to use CLI `setReplication`
* Various small text tuning in Alluxio storage page

### CODE REVIEW

File: docs/en/basic/Command-Line-Interface.md [hunk 1]
```diff
@@ -695,6 +695,23 @@ For example, `setfacl` can be used to give read and execute permissions to a use
 $ ./bin/alluxio fs setfacl -m "user:testuser:r-x" /testdir/testfile
 ```
 
+### setReplication
+
+The `setReplication` command sets the max and/or min replication level of a file or all files under
+a directory recursively. This is a metadata operation and will not cause any replication to be
+created or removed immediately. The replication level of the target file or directory will be
+changed automatically in background. This command takes an argument of `--min` to specify the
+minimal replication level and -max for the maximal replication. Specify -1 as the argument of
```
Comment (ZacBlanco) [inline docs/en/basic/Command-Line-Interface.md]: should this be `--max` instead of `-max`?

Comment (apc999) [reply docs/en/basic/Command-Line-Interface.md]: good  catch!

**Comment by apc999** [comment]
PTAL

File: docs/en/advanced/Alluxio-Storage-Management.md [hunk 2]
```diff
@@ -412,6 +414,74 @@ $ bin/alluxio runTests -Dalluxio.user.file.create.ttl=3m -Dalluxio.user.file.cre
...
+It is possible that in the same file different blocks have a different number
+of replicas according to the popularity.
+
+By default, this replication or eviction decision and the corresponding data transfer is completely
+transparent to users and applications accessing data stored in Alluxio.
+
+### Active Replication
+
+In addition to the dynamic replication adjustment, Alluxio also provides APIs and command-line
+interfaces for users to maintain a target range of replication level for a file explicitly.
```
Comment (ZacBlanco) [inline docs/en/advanced/Alluxio-Storage-Management.md]: 1.) replication level**s**
2.) I think it sounds better if we say "...users to **explicitly** maintain a target range of replication level**s** for a file" or we could just drop "explicitly"

**Comment by apc999** [comment]
@ZacBlanco comments addressed, PTAL

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
@calvinjia  please cherry-pick this PR to branch-1.8

**Pull request merged**